### DISCUSSION TITLE
Spawning language server with cmd: `zk` failed.

### DISCUSSION THREAD

**Discussion by caizoryan** [zk-org/zk-nvim#99] [Help] [answered]
Hey I'm not sure what is causing this issue but I was trying to setup and this error keeps coming up whenever I try to do something. 

![Screenshot 2023-02-06 at 1 40 26 PM](https://user-images.githubusercontent.com/13258280/217057258-f7c33980-36d3-4ef0-b09a-6f86f2d87722.png)

**Comment by mickael-menu** [comment, marked as answer]
Did you install [`zk`](https://github.com/mickael-menu/zk) on your system? What does `where zk` give?

**Reply by caizoryan** [reply]
I just realized I didn't. That solved the problem thanks!
